packages/core/src/child-proxy/child-process-proxy-worker.ts

```diff
@@ -26,7 +26,11 @@ export class ChildProcessProxyWorker {
       process.send(str);
     }
   }
-  private handleMessage(serializedMessage: string) {
+  private handleMessage(serializedMessage: unknown) {
+    if (typeof serializedMessage !== 'string') {
```

Comment:
Maybe its better to throw an error? Or use `typeof serializabedMessage === 'string'? serializedMessage: String(serializedMessage)`

Or simply use `serializedMessage as string` where we use it, since we're sure its a string anyway